lang/python/django-constance/Makefile

```diff
@@ -9,59 +9,64 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=django-constance
 PKG_VERSION:=2.3.1
-PKG_RELEASE:=3
-PKG_LICENSE:=BSD-3-Clause
+PKG_RELEASE:=4
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/d/django-constance
 PKG_HASH:=a49735063b2c30015d2e52a90609ea9798da722ed070f091de51714758a5d018
+
 PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-django-constance-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-$(PKG_NAME)-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
 include ../python3-package.mk
 
 PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
 
-define Package/python-django-constance/Default
+define Package/$(PKG_NAME)/Default
```

Comment:
There's an old rule about not using `$(PKG_NAME)` in `call`, `define`, and `eval` lines.  The `PKG_.*` lines above are excempt.  Its tersely written in https://openwrt.org/docs/guide-developer/packages:
>
>Avoid reuse of PKG_NAME in call, define and eval lines for consistency and readability. Write the full name instead.

See #4119
I have seen this rule broken so many times recently, and also showing up in the main repo, that I'm not sure it is still being followed; however, since this is already abiding by the rule, I wouldn't change them here.

Reply:
> I have seen this rule broken so many times recently, and also showing up in the main repo, that I'm not sure it is still being followed; however, since this is already abiding by the rule, I wouldn't change them here.

ack
wasn't aware of it;
good to know; 

since that is a rule, i'll probably clean-up my packages in future changes;
the reason i used it, is because there were many duplicate changes during `django` packages conversions to python3;
and i used one change that i thoroughly reviewed and diffed the others against the first;

when python package variants go away, things should be easier to maintain;
there are so many places to error-out due to duplication

and i even thought about maybe doing a new `django.mk` or `python-package-defaults.mk` that just adds both python & python3 build rules for a package, but it's not worth it[because python will go away]